client/src/components/TextInputComponent/TextInput.stories.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+
+import TextInput from "./TextInput";
+
+storiesOf("TextInput", module).add("Primary", () => (
+  <TextInput label={"Sign up here"}></TextInput>
```

Comment:
I don't think you need the brackets here

Reply:
You have this with opening and closing tags - can you just do a self-closing tag?

Reply:
Also, it looks like this `label` property is controlling the placeholder text and not the label for these inputs that's in the mock (the text above the input itself). Can you maybe rename this `label` prop to `placeholder`, so it more semantically describes what it does, and add a `label` prop for the label above the input?